almond/rootfs/etc/services.d/almond/run

```diff
@@ -7,6 +7,13 @@ export THINGENGINE_BASE_URL
 # Set the Ingress URL as Almond base URL for correct handling
 THINGENGINE_BASE_URL=$(bashio::addon.ingress_entry)
 
+mkdir -p /data/almond-server
+if [ ! -f /data/almond-server/prefs.db ]; then
+    # Skip authentication handling
+    echo '{"server-login":{"password":"x","salt":"x","sqliteKeySalt":"x"}}' \
+      > /data/almond-server/prefs.db
```

Comment:
If this file contains fixed content, it can be added as a fixed file. It doesn't need to be "created" from a bash file.

Additionally, this is not runtime/service part, but an init part as it seems. Please put this into an init script.

Reply:
IIRC there was a race condition during initialization if this was done in the init script. Also, it cannot be a fixed file because it lives in the data partition.

Reply:
Alternatively, store a file outside of the data folder and move it into place.

Reply:
This comment has not been addressed yet.